app/views/taxons/_form_fields.html.erb

```diff
@@ -1,3 +1,22 @@
+<%= f.input :path_prefix,
+  input_html: { class: 'js-path-prefix form-control' },
+  label: I18n.t('views.taxons.path_prefix'),
+  collection: path_prefixes_for_select %>
+
+<%= f.input :path_slug,
+  input_html: { class: 'js-path-slug form-control' },
+  label: I18n.t('views.taxons.path_slug') %>
+
+<%= f.input :base_path,
```

Comment:
To prevent confusion, if you're never expecting this field to be submitted, it might be better to give it another name and/or not use the form helper like this.